workflow-templates/im-deploy-on-prem-website.yml

```diff
@@ -1,4 +1,4 @@
-# Workflow Code: FuzzyDragon_v26    DO NOT REMOVE
+# Workflow Code: FuzzyDragon_v27    DO NOT REMOVE
```

Comment:
This action name should probably be changed to remove the "on-prem" part. Maybe `im-deploy-iis-website` is better? Or add something about deploying to a _domain-joined_ IIS server.